src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -16,15 +16,19 @@ public class CommandResult {
     /** Help information should be shown to the user. */
     private final boolean showHelp;
 
+    /** Current groups should be shown to the user. */
+    private final boolean showGroups;
+
     /** The application should exit. */
     private final boolean exit;
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean showGroups, boolean exit) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
+        this.showGroups = showGroups;
```

Comment:
could be an object for easier understanding? was confused when used this was created in other classes